src/game.cpp

```diff
@@ -9976,8 +9977,9 @@ void game::plthrow(int pos)
     temp_exit_fullscreen();
     m.draw( w_terrain, u.pos() );
 
-    // pl_target_ui() sets x and y, or returns empty vector if we canceled (by pressing Esc)
-    std::vector<tripoint> trajectory = pl_target_ui( TARGET_MODE_THROW, &thrown, range );
+    // target_ui() sets x and y, or returns empty vector if we canceled (by pressing Esc)
+    std::vector<tripoint> trajectory;
+    trajectory = target_handler().target_ui( u, TARGET_MODE_THROW, &thrown, range );
```

Comment:
This is fear of the astyler, the guide suggests line length less than 100.
With an alias for trajectory's return type and `target_handler` a member of `player` it can be more concise, something like:
`line3d trajectory = pc.thandler.target_ui( args );`